src/pages/public_registration/DeletedPublicationInformation.tsx

```diff
@@ -1,30 +1,76 @@
 import { Registration, RegistrationStatus } from '../../types/registration.types';
-import { Box, Typography } from '@mui/material';
+import { Box, Link, Typography } from '@mui/material';
 import { useTranslation } from 'react-i18next';
+import { getTitleString } from '../../utils/registration-helpers';
+import { useQuery } from '@tanstack/react-query';
+import { fetchRegistration } from '../../api/registrationApi';
+import { getIdentifierFromId } from '../../utils/general-helpers';
+import { PageSpinner } from '../../components/PageSpinner';
 
 interface DeletePublicationInformationProps {
   registration: Registration;
 }
 
+function extractOriginalIdentifier(registration: Registration): string | null {
+  return registration.duplicateOf ? getIdentifierFromId(registration.duplicateOf) : null;
+}
+
 export const DeletedPublicationInformation = ({ registration }: DeletePublicationInformationProps) => {
   const { t } = useTranslation();
+
+  const originalIdentifier = extractOriginalIdentifier(registration);
```

Comment:
like greit og bare ta det sånn?

```suggestion
export const DeletedPublicationInformation = ({ registration }: DeletePublicationInformationProps) => {
  const { t } = useTranslation();

  const originalIdentifier = getIdentifierFromId(registration.duplicateOf);
```